src/bin/cql-stress-cassandra-stress/java_generate/values/decimal.rs

```diff
@@ -0,0 +1,33 @@
+use bigdecimal::FromPrimitive;
+use scylla::frame::response::result::CqlValue;
+
+use crate::java_generate::distribution::Distribution;
+
+use super::{ValueGenerator, ValueGeneratorFactory};
+
+#[derive(Default)]
+pub struct Decimal;
+
+impl ValueGenerator for Decimal {
+    fn generate(
+        &mut self,
+        identity_distribution: &mut dyn Distribution,
+        _size_distribution: &mut dyn Distribution,
+    ) -> CqlValue {
+        CqlValue::Decimal(
+            // unwrap is safe since BigDecimal::from_i64 returns Some().
+            bigdecimal::BigDecimal::from_i64(identity_distribution.next_i64())
+                .unwrap()
+                .try_into()
+                .expect("Failed to convert i64 to CqlDecimal."),
+        )
+    }
+}
```

Comment:
Do we really have to pull in `bigdecimal` dependency for that?
`CqlDecimal` has a constructor: `from_signed_be_bytes_slice_and_exponent` - perhaps it could suffice?

Reply:
Tests are missing, again.

Reply:
> Do we really have to pull in `bigdecimal` dependency for that? `CqlDecimal` has a constructor: `from_signed_be_bytes_slice_and_exponent` - perhaps it could suffice?

I removed the dependency and used the constructor you mentioned. Added a comment that justifies values provided to the constructor.